site/en/docs/extensions/mv3/hosting/index.md

```diff
@@ -2,68 +2,50 @@
 layout: "layouts/doc-post.njk"
 title: "Chrome Web Store"
 date: 2012-09-18
-updated: 2018-06-12
+updated: 2021-12-10
 description: >
-  How to host your extension in the Chrome Web Store and update an extension
-  that's hosted in the Chrome Web Store.
+  How to publish and update your extension in the Chrome Web Store.
 ---
 
-{% include 'partials/extensions/mv2page-in-mv3.md' %}
+<!-- This docs has been changed to primarily explain 
+     how to publish and update to the Chrome Web Store.
+     TODO: A rewrite should explore examples and use cases of
+     each distribution options (locally unpacked, enterprise, 
+     JSON preference file, personal server for Linux users). -->
 
-Most extensions are hosted in the [Chrome Web Store][1] to best [protect users from malicious
-extensions][2].
+Most extensions are hosted in the [Chrome Web Store][what-is-cws], an online marketplace where users
+can browse for Chrome extensions and themes.
 
-## Hosting {: #hosting }
+There are three exceptions to the Chrome Web Store hosting rule:
 
-All extensions are distributed to users as a special ZIP file with a `.crx` suffix. Extensions
-hosted in the [Chrome Web Store][3] are uploaded through the [Developer Dashboard][4] as `.zip`
-files. The [publishing][5] process automatically converts the `.zip` into a `.crx` file.
+1.  Extensions that are distributed through the [enterprise policy][enterprise-policy].
+1.  Unpacked extension directories from a local machine while in [developer mode][load-unpacked].
```

Comment:
```suggestion
1.  Unpacked extension directories from a local machine while in [developer mode][load-unpacked].
1.  Extensions that are distributed through the [enterprise policy][enterprise-policy].
```

Nit: flip the order of these as developer mode is probably the most common option our readers will encounter.

Reply:
Good point! 🙂